examples/snapshot-testing/test.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Same here. It complains about `React` and `HelloWorld`. Although `HelloWorld` is used. Probably because the extension is not `jsx`?